src/config.ts

```diff
@@ -52,16 +52,6 @@ class BaseConfiguration {
    */
   minCacheSideLimit = 256;
 
-  /**
-   * When 'true', style information is not retained when copy/pasting text, making
-   * pasted text use destination style.
-   * Defaults to 'false'.
-   * @type Boolean
-   * @default
-   * @deprecated
-   */
-  disableStyleCopyPaste = false;
```

Comment:
If we want to preserve this functionality I think it is best to add an option to `IText` 
`patseOptions: source-formatting` | `destination-formatting` or something